### DISCUSSION TITLE
[Question] Struggling to Download Auto Ortho

### DISCUSSION THREAD

**Discussion by maxhaskell20** [kubilus1/autoortho#544] [Q&A]
### Please check existing docs and issues first.

- [X] I have checked for existing answers
- [X] This is not a bug or other "Issue"
- [X] I have read the FAQ and documentation

### System Details

-AutoOrtho_win_0.7.2
-Scenery Pack: North America
-Windows 11 x64

### question

Hello,

Hoping someone can help me here. I'm having quite a bit of trouble downloading AutoOrtho, I have downloaded and tried to reinstall several times. When I tried to most recently download it, this is the error message I recieved. It downloads really well typically until it get to 35_zip and then all of the sudden it stops downloading. Could someone potentially help me with what to do because I am stuck here. Any help is greatly appreciated and thank you in advance. Attached is the most recent error message I received. 
![2024-10-17](https://github.com/user-attachments/assets/2a8fa60e-c87b-4f18-a2b4-e2c7642ea03d)

**Comment by Uemmet** [comment]
i have the same problem
